app/assets/javascripts/intl-tel-input.js.erb

```diff
@@ -0,0 +1,22 @@
+// Setting hidden input for intl code
+
+var countryData = $.fn.intlTelInput.getCountryData(),
+  telInput = $("#user_phone_form_phone");
+
+
+// initialise plugin
+telInput.intlTelInput({
+  utilsScript: "<%= asset_path 'intl-tel-number/utils.js' %>",
+  preferredCountries: ["us", "ca"]
+
+});
+
+// set it's initial value
+var initialCountry = telInput.intlTelInput("getSelectedCountryData").dialCode;
+telInput.val("+" + initialCountry);
+
+// inserting country code on dropdown change
+telInput.on("countrychange", function(e, countryData) {
+  $('#user_phone_form_phone').val("+" + countryData.dialCode);
+  $('#user_phone_form_international_code').val(countryData.iso2.toUpperCase());
+});
```

Comment:
This file should live somewhere in `app/javascript/app`, in either `components`, `modules`, or `utils`.

Reply:
Webpacker no longer uses the `app/assets/javascripts` directory. I see that we have one file in there, but it should probably be moved.

Reply:
try `javascript_pack_tag`

Reply:
I think in this case, at least one of the files would have to go in `app/javascript/packs`. I think the way JS is organized is that packs are the main JS files that are included either everywhere, like `application.js`, or only on certain pages. Those files then in turn import functionality from other files that are in components, modules, or utils.